src/types.ts

```diff
@@ -15,204 +15,71 @@ export interface Node extends swc.Node {
    */
   span?: swc.Span
 }
-export type AggregateType = {
-  ClassMember: swc.ClassMember
-  Declaration: swc.Declaration
-  DefaultDecl: swc.DefaultDecl
-  ExportSpecifier: swc.ExportSpecifier
-  Expression: swc.Expression
-  ImportSpecifier: swc.ImportSpecifier
-  JSXAttrValue: swc.JSXAttrValue
-  JSXElementChild: swc.JSXElementChild
-  JSXElementName: swc.JSXElementName
-  Literal: swc.Literal
-  ModuleDeclaration: swc.ModuleDeclaration
-  ObjectPatternProperty: swc.ObjectPatternProperty
-  Pattern: swc.Pattern
-  Property: swc.Property
-  PropertyName: swc.PropertyName
-  Statement: swc.Statement
-  TsFnParameter: swc.TsFnParameter
-  TsParameterPropertyParameter: swc.TsParameterPropertyParameter
-  TsType: swc.TsType
-  TsTypeElement: swc.TsTypeElement
-}
+
+type AggregatedNode =
+  | swc.ClassMember
+  | swc.Declaration
+  | swc.DefaultDecl
+  | swc.ExportSpecifier
+  | swc.Expression
+  | swc.ImportSpecifier
+  | swc.JSXAttrValue
+  | swc.JSXElementChild
+  | swc.JSXElementName
+  | swc.Literal
+  | swc.ModuleDeclaration
+  | swc.ObjectPatternProperty
+  | swc.Pattern
+  | swc.Property
+  | swc.PropertyName
+  | swc.Statement
+  | swc.TsFnParameter
+  | swc.TsParameterPropertyParameter
+  | swc.TsType
+  | swc.TsTypeElement
+
 export type AnyNode =
-  | swc.ArrayExpression
```

Comment:
note that all removed types form `AnyNode` is provided by on of the `AggregatedNode` and will not be lost